build/Targets/VSL.Versions.targets

```diff
@@ -17,6 +17,18 @@
     <BuildNumberBuildOfTheDayPadded Condition="('$(BuildNumber)' != '') AND ($(BuildNumber.Split('.').Length) == 2)">$(BuildNumber.Split('.')[1].PadLeft(2,'0'))</BuildNumberBuildOfTheDayPadded>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
Nit: Roslyn declares all of these in a Dependencies.props, it would be great if this was consistent across repos.